package.json

```diff
@@ -2994,7 +2994,7 @@
         "@microsoft/vscode-azext-azureutils": "^1.1.5",
         "@microsoft/vscode-azext-utils": "^1.2.2",
         "dayjs": "^1.11.7",
-        "dockerfile-language-server-nodejs": "^0.9.0",
+        "dockerfile-language-server-nodejs": "^0.10.0",
```

Comment:
I'm still hitting the `textDocument/semanticTokens/full` failure

Reply:
Unfortunately, I am unable to reproduce this problem when I launch the second instance of VS Code. 🤔

Reply:
I'll try restarting completely...

Reply:
Ugh, still happening after I rebuilt the extension into a VSIX and installed it. 😕

[Error - 4:29:50 PM] Request textDocument/semanticTokens/full failed.
  Message: Request textDocument/semanticTokens/full failed with message: Maximum call stack size exceeded
  Code: -32603

Reply:
https://github.com/microsoft/vscode-docker/assets/15629116/e196bf06-27e4-4ebb-83cb-7b43e415b1a0

My Extension Development Host window is working well for me.

Reply:
Could it be OS somehow? I'm on Win11

Reply:
Potentially related, if I change the line endings from CRLF to LF in the bottom, the error also goes away. AFAIK there are not actually any carriage returns in the text; `\r` has no results in a regex search.

Reply:
> Oh, even weirder, if line 5 is indented by even one space, I do not have the error.

Yeah. That is expected because the problem is that it's a "multiline" argument. The space will break it apart so the parsing doesn't cause any problems.

> Potentially related, if I change the line endings from CRLF to LF in the bottom, the error also goes away.

Thanks for noticing this. I can reproduce the problem after switching to CRLF. :) Will work on a fix.

Reply:
@bwateratmsft I've bumped it to version 0.10.2 which should fix the CRLF issue.